tests/browsing_context/test_traverse_history.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2023 Google LLC.
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from test_helpers import execute_command, goto_url
+
+
+@pytest.mark.asyncio
+async def test_traverse_history(websocket, context_id, html):
+    await assert_href(websocket, context_id, "about:blank")
+
+    await goto_url(websocket, context_id, html())
+    await assert_href(websocket, context_id, "data:text/html,")
+
+    await execute_command(
+        websocket, {
+            "method": "browsingContext.traverseHistory",
+            "params": {
+                "context": context_id,
+                "delta": -1
+            }
+        })
+    await assert_href(websocket, context_id, "about:blank")
+
+
+async def assert_href(websocket, context_id, href):
```

Comment:
nit: `assert_href_equals` would be more readable. When I see `assert_href` I expect a single argument ("assert this thing is an href").